packages/koa-shopify-webhooks/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ### Changed
 
+- Reorder exports map to prioritize the `esnext` condition. [[#2148](https://github.com/Shopify/quilt/pull/2148)]
```

Comment:
Might need to rebase, as I think @vsumner released a new version of this package this morning.

Reply:
Fixed 👍